src/styles/blocks/card.scss

```diff
@@ -0,0 +1,109 @@
+@import "../utils/variables";
+
+.card {
+  display: flex;
+  flex-direction: column;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  width: $card-width;
+
+  &__image {
+    height: 134px;
+    width: 160px;
+    margin: 0 auto 40px;
+  }
+
+  &__name {
+    margin: 0 0 4px;
+
+    &--text_medium {
+      @extend %custom-font;
+      font-size: $font-size-medium;
+      font-weight: $font-weight-medium;
+      line-height: $line-height-text-medium;
+      color: $color-text-primary;
+    }
+  }
+
+  &__code {
+    margin: 0 0 16px;
+
+    &--text_small {
+      @extend %custom-font;
+      font-size: 10px;
+      font-weight: $font-weight-less;
+      line-height: 14px;
+      color: $color-text-light-grey;
+    }
+  }
+
+  &__rating {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+
+    margin-bottom: 24px;
+  }
+
+  &__review {
+    margin: 0;
+
+    &--text_small {
+      @extend %custom-font;
+      font-size: 10px;
+      font-weight: $font-weight-less;
+      line-height: 14px;
+      color: $color-text-primary;
+    }
+  }
+
+  &__buy {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    width: 166px;
+    height: 40px;
```

Comment:
I think if it's set as a block, you don't need width

Reply:
Done.